InternxtDesktop/Views/Backups/FolderSelectorView.swift

```diff
@@ -103,8 +103,10 @@ struct FolderSelectorView: View {
                 closeWindow()
                 try await self.backupsService.startBackup( onProgress: {progress in
                 })
+                EventsUtils.trackSuccessBackup(foldersToBackup: backupsService.foldersToBackup.count)
```

Comment:
Is better to track this from the XPCBackupService, rather than from the client app